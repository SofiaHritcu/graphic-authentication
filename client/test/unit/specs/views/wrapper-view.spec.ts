import { shallowMount } from "@vue/test-utils";
import WrapperView from "@/views/wrapper-view.vue";


describe("Wrapper View", () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallowMount(WrapperView);
  });

  afterEach(() => {
    wrapper.destroy();
  });

  describe("snapshot", () => {
    it("should match snapshot", () => {
      expect(wrapper.vm.$el).toMatchSnapshot();
    });
  });
});
