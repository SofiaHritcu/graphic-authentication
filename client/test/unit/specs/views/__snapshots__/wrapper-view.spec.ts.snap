// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper View snapshot should match snapshot 1`] = `
<v-container
  class="ga__wrapper-view__container mt-10"
  fluid=""
>
  <router-view />
</v-container>
`;
